src/guides/v2.3/frontend-dev-guide/layouts/layout-create.md

```diff
@@ -0,0 +1,36 @@
+---
+group: frontend-developer-guide
+title: Create a new layout
+functional_areas:
+  - Frontend
+---
+
+### Create a new page layout in custom theme
```

Comment:
Headers need to go in order. The title is # so the next header should be ## and so forth.

Reply:
@dobooth Changed the header section and commit the code changes. Please review it.
Thanks for the review.